### DISCUSSION TITLE
Watch configmap not owned by CR and in a diff namespace

### DISCUSSION THREAD

**Discussion by pbikki** [kubernetes-sigs/kubebuilder#2870] [Support] [answered]
I'm looking for a way to send a reconcile request to my custom resource (CR) controller when there is an update to a centralized config map (not owned by CR) and located in a different namespace. 

Note: I don't have a reference to the the config map in my CR spec. But I do know where the configmap is. Wondering how I can watch just this specific resource

From the docs, https://book.kubebuilder.io/reference/watching-resources/externally-managed.html
I see that we can achieve something like this by setting up a watch on in the `SetupWithManager`. However I'm not sure how can we achieve this when CR and configmap are in different namespaces and when I don't have a reference to the configmap in my CR

**Comment by FillZpp** [comment, marked as answer]
@poojitha-bikki  You can watch configmaps with your custom handler, in which you find the CR that should be triggered by this changed configmap and put the CR request into workqueue.

**Comment by camilamacedo86** [comment]
Closing this one
However, if you see that is required please feel free to reopen
